ReactAndroid/src/main/java/com/facebook/react/uimanager/JSTouchDispatcher.java

```diff
@@ -30,6 +30,7 @@
   private int mTargetTag = -1;
   private final float[] mTargetCoordinates = new float[2];
   private boolean mChildIsHandlingNativeGesture = false;
+  private long mGestureStartTime;
```

Comment:
Finger2 will trigger `ACTION_POINTER_DOWN`, not `ACTION_DOWN`. Only the first finger can trigger `ACTION_DOWN`. It is actually the same gesture when there are two different fingers. 
> - `ACTION_DOWN`—For the first pointer that touches the screen. This starts the gesture. The pointer data for this pointer is always at index 0 in the MotionEvent.
> - `ACTION_POINTER_DOWN`—For extra pointers that enter the screen beyond the first. The pointer data for this pointer is at the index returned by getActionIndex().
> - `ACTION_MOVE`—A change has happened during a press gesture.
> - `ACTION_POINTER_UP`—Sent when a non-primary pointer goes up.
> - `ACTION_UP`—Sent when the last pointer leaves the screen.